x11-packages/sdl2/configure.patch

```diff
@@ -19,8 +21,10 @@
 -$as_echo "#define SDL_VIDEO_DRIVER_ANDROID 1" >>confdefs.h
 -
 -                    SUMMARY_video="${SUMMARY_video} android"
+-                    have_video=yes
 -                fi
 -                ;;
++        case "$host" in 
```

Comment:
I think you just added an extra space at the end of this line, seems like a spurious change.

Reply:
ah, right